ocs_ci/framework/conf/default_config.yaml

```diff
@@ -37,7 +37,7 @@ REPORTING:
 # This is the default information about environment. Will be overwritten with
 # --cluster-conf file.yaml data you will pass to the pytest.
 ENV_DATA:
-  cluster_name: "ocs-ci"  # needs to be changed in ocscilib plugin
+  cluster_name: null  # will be changed in ocscilib plugin
```

Comment:
we should still keep "ocs-ci" as prefix and check if the string has changed(in case user modified)

Reply:
This is now done in `ocscilib.py`